example_notebooks/example_scripts/example_cp.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Example of CellProfiler project for idrstream
+
+# ## Import Libraries
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+import shutil
+
+from cellpose import core
+
+import sys
+sys.path.append("../")
+from idrstream.CP_idr import CellProfilerRun
+
+
+# ## Initialize idrstream
+
+# In[2]:
+
+
+pipeline_path = pathlib.Path("../example_files/CP_files/mitocheck_idr_cp.cppipe")
+# need to fill in on fig
+plugins_directory = pathlib.Path("/home/jenna/Desktop/Github/CellProfiler/cellprofiler/modules/plugins")
+idr_id = "idr0013"
+tmp_dir = pathlib.Path("../tmp/")
+final_data_dir = pathlib.Path("../mitocheck_control_features/CP_features")
+try:
+    shutil.rmtree(tmp_dir)
+    # uncomment the line below if you would like to remove the final data directory (e.g. all .csv.gz files)
+    # shutil.rmtree(final_data_dir)
+    pass
+except:
+    print("No files to remove")
+
+stream = CellProfilerRun(pipeline_path, plugins_directory, idr_id, tmp_dir, final_data_dir, log='cp_idrstream.log')
+
+
+# ## Initialize CellProfiler metadata compiler
+
+# In[3]:
+
+
+data_to_process_tsv = pathlib.Path("../example_files/data_to_process.tsv")
+metadata_save_path = pathlib.Path("../example_files/data_to_process.csv")
+
+data_to_process = stream.convert_tsv_to_csv(data_to_process_tsv, metadata_save_path)
```

Comment:
```suggestion
stream.convert_tsv_to_csv(data_to_process_tsv, metadata_save_path)
```
This method does not return anything so setting a variable equal to its output just makes `data_to_process = None`

Reply:
:+1: